src/openforms/emails/models.py

```diff
@@ -38,7 +38,8 @@ class ConfirmationEmailTemplate(models.Model):
                 required_template_tags=[
                     "appointment_information",
                     "payment_information",
-                ]
+                ],
+                backend="openforms.template.openforms_backend",
```

Comment:
... I feel these validators are a kind of typeguard, and it doesn't sit quite right that on use of the value, we could specify a different backend.

Reply:
They are here to prevent badly formatted (syntax errors) in templates submitted by form builders to end up in the system which then generates runtime errors and clutters error monitoring.

It's not a bug we (developers) can or should fix, but it is up to us to provide the tooling/feedback to users so they can correct their own mistakes. It's definitely going to cause problems if we render with a stricter subset of the language than that we validate with.